wasmbinding/stargate_bindings.go

```diff
@@ -0,0 +1,76 @@
+package wasmbinding
+
+import (
+	"sync"
+
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/auth"
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/bank"
+	distr "github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/distribution"
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/feegrant"
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/mint"
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/slashing"
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/staking"
+)
+
+// StargateLayerBindings keeps whitelist and its deterministic
+// response binding for stargate queries.
+//
+// The query can be multi-thread, so we have to use
+// thread safe sync.Map instead map[string]bool.
```

Comment:
It seems that the value in this map is not `bool` but the binding itself. If that's the case, can we avoid mentioning `map[string]bool` or change the value, please? I got stumbled by this for a sec

Reply:
Ah my bad, fixed!